src/graphics/mod.rs

```diff
@@ -290,10 +285,7 @@ impl BackendSpec for GlBackendSpec {
         color_format: gfx::format::Format,
         depth_format: gfx::format::Format,
         window: &glutin::WindowedContext,
-    ) -> Option<(
-        gfx::handle::RawRenderTargetView<Self::Resources>,
-        gfx::handle::RawDepthStencilView<Self::Resources>,
-    )> {
+    ) -> MainTargetView<Self::Resources> {
         // Basically taken from the definition of
         // gfx_window_glutin::update_views()
         let dim = color_view.get_dimensions();
```

Comment:
If you check the `if let` statement below this line you'll notice that it can be removed and just call `update_views_raw`. Please let me know if this change is desirable.